### ISSUE TITLE
Ports with complex markup - only one port is responding

### ISSUE THREAD

**Issue by shukiAnavi** [clientIO/joint#990]
Hi!

I added two 'outPorts' to my shape and it works fine when the markup looks like that:

`
            decisionCell.portProp('no', 'markup',
                '<circle r="19" fill="#FFFFFF" stroke="#4B525C" stroke-width="2" cx="-116" cy="11"/>');
`

`
            decisionCell.portProp('yes', 'markup',
                '<circle r="19" fill="#FFFFFF" stroke="#4B525C" stroke-width="2" cx="-10" cy="-24"/>');
`

This is the result:
![image](https://user-images.githubusercontent.com/10337876/47334858-24c08980-d691-11e8-8cde-af7bddace1be.png)

And it works fine when I'm stretching the links from both ports.

The problem occurring when I'm changing the markups to look like that:

`
decisionCell.portProp('no', 'markup',
                '<g><circle r="20" fill="#4B525C" cx="-100" cy="27"/><image class="port-body" xlink:href="images/automation/thumbs-down-white.svg" x="-116" y="11" height="32px" width="32px"/></g>');
`

`
decisionCell.portProp('yes', 'markup',
                '<g><circle r="19" fill="#FFFFFF" stroke="#4B525C" stroke-width="2" cx="-10" cy="-24"/><image class="port-body" xlink:href="images/automation/thumbs-up.svg" x="-26" y="-40" height="32px" width="32px"/></g>');
`

I added and image for each port and wrap the image and the circle with `<g>` element.

This is the result:
![image](https://user-images.githubusercontent.com/10337876/47335078-0eff9400-d692-11e8-8dc8-26c5aa850755.png)

Now when I'm stretching the first link from any of the ports and connect it to another shape, everything works fine.
When I'm trying to do the same thing on the other port, there is no link and the shape starts to respond to the dragging event.

Any ideas for why that happens?

**Comment by kumilingus** [comment]
It seems like you were setting `magnet: true` attribute on a circle element. When you changed the markup, you hid the circle with an image (user can't reach the circle with a pointer). You can try setting `pointer-events: none` on the image element.

```js
decisionCell.portProp('no', 'markup', '<g><circle r="20" fill="#4B525C" cx="-100" cy="27"/><image class="port-body" pointer-events="none" xlink:href="images/automation/thumbs-down-white.svg" x="-116" y="11" height="32px" width="32px"/></g>');
```

**Comment by shukiAnavi** [comment]
Thanks for the response.
Unfortunately, that does not solve the problem.

When I added the:
`pointer-events:none`
None of the ports can be linked.

In the original question, both of the ports were active when not connected to other elements.
Once I link one of them to an element, the other port is cannot be linked.

In this example, I successfully connected the 'thumb-up' port to another:

![image](https://user-images.githubusercontent.com/10337876/47414122-e4d4d180-d778-11e8-8802-67d92d6dca6e.png)

When the linking that described in the image above is done,
the 'thumb-down' port will be no longer act as an output port.
